Cargo.toml

```diff
@@ -1,8 +1,14 @@
 [package]
 name = "dapr"
-version = "0.2.0"
+version = "1.0.0"
```

Comment:
1.0 seems pretty aggressive. I was under the impression that this crate was still very much a work in progress.

Reply:
Agree. Should this match the version for dapr runtime?